dev/run-tests.py

```diff
@@ -180,9 +180,9 @@ def run_scala_style_checks():
     run_cmd([os.path.join(SPARK_HOME, "dev", "lint-scala")])
 
 
-def run_java_style_checks():
+def run_java_style_checks(build_profiles):
```

Comment:
and .. let's also leave a comment saying, "The same profiles used for building are used to run Checkstyle by SBT as well because the previous build looks reused for Checkstyle and affecting Checkstyle. See SPARK-27130."

Reply:
How about?:
```python
def run_java_style_checks(build_profiles):
    set_title_and_block("Running Java style checks", "BLOCK_JAVA_STYLE")
    # The same profiles used for building are used to run Checkstyle by SBT as well because
    # the previous build looks reused for Checkstyle and affecting Checkstyle. See SPARK-27130.
    profiles = " ".join(build_profiles)
    print("[info] Checking Java style using SBT with these profiles: ", profiles)
    run_cmd([os.path.join(SPARK_HOME, "dev", "sbt-checkstyle"), profiles])
```
![image](https://user-images.githubusercontent.com/5399861/54196515-9c863800-44fc-11e9-866d-dbd42de99c45.png)

Reply:
Yea cool